apps/wallet/tests/sites-to-cs-messaging.spec.ts

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type Page } from '@playwright/test';
+
+import { expect, test } from './fixtures';
+import { createWallet } from './utils/auth';
+
+function getInAppMessage(page: Page, id: string) {
+    return page.evaluate(
+        (anId) =>
+            Promise.race([
+                new Promise((r) => setTimeout(() => r(null), 1000)),
```

Comment:
Why resolve here instead of reject?

Reply:
I think it's because this is for unknown request cases, but for those I'd either:
1. expect a rejection
2. update the code to respond to unknown commands